modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,194 @@
+
+* With this update, you can define and use a `PipelineRun` context variable in the `pipelineRun.workspaces.subPath` configuration.
+
+* With this update, the `ClusterResolver`, `BundleResolver`, `HubResolver`, and `GitResolver` features are now available by default. 
++
+[NOTE]
+====
+The `taskref.bundle` and `pipelineref.bundle` bundles are deprecated and will be removed in a future release.
+====
+
+[id="triggers-new-features-1-11_{context}"]
+=== Triggers
+
+* With this update, Tekton Triggers support the `Affinity` and `TopologySpreadConstraints` values in the `EventListener` specification. You can use these values to configure Kubernetes and custom resources for an `EventListener` object.
+* This update adds a Slack interceptor that allows you to extract fields by using a slash command in Slack. The extracted fields are sent in the form data section of an HTTP request.
+
+
+[id="operator-new-features-1-11_{context}"]
+=== Operator
+
+* With this update, you can configure pruning for each pipeline run or task run resource by setting a boolean field named `prune-per-resource` in the `TektonConfig` CR. You can also configure pruning for each pipeline run or task run resource in a  namespace by adding the annotation, `operator.tekton.dev/prune.prune-per-resource=true`, to that namespace.
+
+* With this update, if there are any changes in the {product-title} cluster-wide proxy, Operator Lifecycle Manager (OLM) recreates the {pipelines-title} Operator. 
+* With this update, you can disable the pruner feature by setting the `config.pruner.disabled` field to `true` in the `TektonConfig` CR.
+
+
+[id="chains-new-features-1-11_{context}"]
+=== {tekton-chains}
+
+* With this update, {tekton-chains} is now generally available for use.
+
+* With this update, you can use the skopeo tool with {tekton-chains} to generate keys, which are used in the `cosign` signing scheme. 
+
+
+[id="tekton-hub-new-features-1-11_{context}"]
+=== {tekton-hub}
+
+:FeatureName: {tekton-hub}
+include::snippets/technology-preview.adoc[]
+
+* This update adds a new endpoint `resource/<catalog_name>/<kind>/<resource_name>/raw` and a new field named `resourceURLPath` in the `resource` API response. This update helps you to obtain the latest raw YAML file of the resource.
+
+[id="tekton-results-new-features-1-11_{context}"]
+=== Tekton Results
+
+:FeatureName: Tekton Results
+include::snippets/technology-preview.adoc[]
+
+* This update adds Tekton Results to the Tekton Operator as an optional component.
+
+
+[id="pac-new-features-1-11_{context}"]
+=== {pac}
+
+* With this update, {pac} allows you to expand a custom parameter within your `PipelineRun` resource by using the `params` field. You can specify a value for the custom parameter inside the template of the `Repository` CR. The specified value replaces the custom parameter in your pipeline run. Also, you can define a custom parameter and use its expansion only when specified conditions were matched for a CEL filter.
```

Comment:
compatible does not seem the correct word here because we use compatibility in terms of software, hardware, or any other platform, not with CEL filter.

Reply:
done